client/src/components/layout/SaveTableForm.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react"
+import translateServerErrors from "../../services/translateServerErrors.js"
+import ErrorList from "./ErrorList.js"
+
+const SaveTableForm = ({ selectedPlayers }) => {
+  const [form, setForm] = useState({
+    title: "",
+    notes: "",
+    players: [],
+  })
+  const [errors, setErrors] = useState([])
+
+  const saveTable = async (event) => {
+    event.preventDefault()
+    const formPayload = { ...form, players: selectedPlayers }
+    
+    try {
+      const response = await fetch(`/api/v1/tables`, {
+        method: "POST",
+        headers: new Headers({
+          "content-Type": "application/json",
+        }),
+        body: JSON.stringify(formPayload),
+      })
+      if (!response.ok) {
+        if (response.status === 422) {
+          const body = await response.json()
+          const newErrors = translateServerErrors(body.errors)
+          return setErrors(newErrors)
+        } else {
+          const errorMessage = `${response.status} (${response.statusText})`
+          const error = new Error(errorMessage)
+          throw error
+        }
+      } else {
+        const body = await response.json()
```

Comment:
not doing anything here with the body after the POST is made. we should probably either update state somewhere with this body info, or redirect to the page where the user can see their new table